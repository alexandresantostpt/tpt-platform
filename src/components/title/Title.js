import PropTypes from 'prop-types'
import React from 'react'
import styled from 'styled-components'

const StyledTitle = styled.h2`
    color: #000000;
    font-size: 1.875rem;
    font-weight: 300;
    margin-bottom: 1.0675rem;
    opacity: 0.67;
    text-transform: uppercase;
`

const Title = props => <StyledTitle {...props} />

Title.propTypes = {
    children: PropTypes.string.isRequired
}

export default Title
