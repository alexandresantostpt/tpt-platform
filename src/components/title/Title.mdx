---
name: Title
route: /Title
---
import { Playground, PropsTable } from 'docz'

import Title from './Title'

import '@cfg/scripts'
import '@cfg/style'

# Title

<PropsTable of={ Title }/>

## Basic Usage

```
<Title>TITLE</Title>
```

<Playground>
    <Title>JAN, 2019</Title>
</Playground>
