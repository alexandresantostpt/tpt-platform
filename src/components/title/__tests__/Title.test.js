import React from 'react'
import renderer from 'react-test-renderer'

import Title from '@components/title/Title'

describe('Tests for Title component', () => {
    it('Should rendering without crash ', () => {
        const tree = renderer.create(<Title>My test</Title>).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
