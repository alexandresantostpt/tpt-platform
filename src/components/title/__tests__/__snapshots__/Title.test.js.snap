// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Title component Should rendering without crash  1`] = `
<h2
  className="Title__StyledTitle-sc-11bxhsf-0 frSblw"
>
  My test
</h2>
`;
